cagov/ui-claim-tracker
# Optionally accept a certificate passphrase

## Ticket

Closes #302 

## Changes

- Adds an optional `PFX_PASSPHRASE` env var

## Testing

`yarn test`

OR

1. Use a certificate that requires a PFX import passphrase
2. Adjust the `P12_FILE` env var in `.env.local` 
3. Add the `PFX_PASSPHRASE` env var in `.env.local`
4. Confirm the certificate works: `yarn dev`
